woocommerce/woocommerce
# Add product pricing block

### All Submissions:






### Changes proposed in this Pull Request:

This creates a pricing block field that could be used for both the list price and the sale price.

Closes #37098 

![Screenshot 2023-03-15 at 2 00 49 PM](https://user-images.githubusercontent.com/2240960/225316344-3f7bdf25-1600-4a95-a7b3-0a44428047ed.png)



### How to test the changes in this Pull Request:




1. Build this branch and enable the product blocks feature flag: `block-editor-feature-enabled`
2. Go to **Products > Add new** and check if the pricing and sales field show up
3. Now create a product using the classic editor and add a price and sale price, load this product in the new product block form, the price and sale price should be displayed correctly.
4. Go to **WooCommerce > Settings** and change the currency
5. Reload the product block editor and make sure the icons in the price field match the new currency.



### Other information:




### FOR PR REVIEWER ONLY:
